---
source: tests/integration_test.rs
expression: "run_code(r#\"\nfunc main() {\n  continue;\n}\n\nmain();\n\"#)"
---
stdout: ""
stderr: ""
error:
  ExecutionError:
    message: Continue must be used in a loop block
    call_stack:
      - main
exit_code: 103
